ngraph/core/include/ngraph/op/gather.hpp

```diff
@@ -61,5 +61,32 @@ namespace ngraph
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
         } // namespace v7
+
+        namespace v8
+        {
+            /// \brief Gather slices from axis of params according to indices
+            class NGRAPH_API Gather : public op::util::GatherBase
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+                Gather() = default;
+
+                /// \param data The tensor from which slices are gathered
+                /// \param indices Tensor with indexes to gather
+                /// \param axis The tensor is a dimension index to gather data from
+                /// \param batch_dims The number of batch dimension in data and indices tensors.
+                Gather(const Output<Node>& data,
+                       const Output<Node>& indices,
+                       const Output<Node>& axis,
+                       const int64_t batch_dims = 0);
+
+                bool visit_attributes(AttributeVisitor& visitor) override;
```

Comment:
Could you add new test on `visit_attributes`?

Reply:
Added. Also bumped to opset8. Please review